app/src/main/java/swen_anigans/mathematicfanatic/Question.java

```diff
@@ -0,0 +1,71 @@
+package swen_anigans.mathematicfanatic;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+
+/**
+ * Created by Matthew Gallagher on 11/14/2016.
+ */
+
+public class Question implements Serializable {
+
+    private int firstNumber;
+    private int secondNumber;
+    private int submittedAnswer;
+    private int answer;
+
+    public Question(){
+        firstNumber = 0;
+        secondNumber = 0;
+        answer = 0;
+        submittedAnswer = 0;
+    }
+
+    public Question(int first, int second, int a){
+        firstNumber = first;
+        secondNumber = second;
+        answer = a;
```

Comment:
i figured by having it send the answer then we could use it for different kinds of math if we ever wanted to.